extensions/mssql/src/config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "1.5.0-alpha.71",
+	"version": "1.5.0-alpha.74",
```

Comment:
Note that we can not checkin with 74 - we will have to wait for next one to come.